spec/support/api/schemas/mixtape.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "http://spotify-apprentice-app.com",
+  "type": "object",
+  "properties": {
+    "mixtape": {
+      "id": "http://spotify-apprentice-app.com/mixtape",
+      "type": "object",
+      "properties": {
+        "songs": {
+          "id": "http://spotify-apprentice-app.com/mixtape/songs",
+          "type": "array",
+          "items": {
+            "id": "http://spotify-apprentice-app.com/mixtape/songs/0",
+            "type": "object",
+            "properties": {
+              "id": {
+                "id": "http://spotify-apprentice-app.com/mixtape/songs/0/id",
+                "type": "string"
+              },
+              "title": {
+                "id": "http://spotify-apprentice-app.com/mixtape/songs/0/title",
+                "type": "string"
+              },
+              "album": {
+                "id": "http://spotify-apprentice-app.com/mixtape/songs/0/album",
+                "type": "string"
+              },
+              "duration": {
+                "id": "http://spotify-apprentice-app.com/mixtape/songs/0/duration",
+                "type": "integer"
+              },
+              "formatted_duration": {
+                "id": "http://spotify-apprentice-app.com/mixtape/songs/0/formatted_duration",
+                "type": "string"
+              },
+              "full_title": {
+                "id": "http://spotify-apprentice-app.com/mixtape/songs/0/full_title",
+                "type": "string"
+              },
+              "artist_id": {
+                "id": "http://spotify-apprentice-app.com/mixtape/songs/0/artist_id",
+                "type": "string"
+              }
```

Comment:
Is `genre_id` returned in this JSON? Doesn't look like it but you might want to add it to the serializer.

Usually when I add a property to a model I also add it to:

- the serializer
- the JSON schema that reference that model
- the factory

Also, now that a song has a genre, you might want to include the serialized genre along with serialized songs.  (Could be separate PR)